ArduCopter/RC_Channel.cpp

```diff
@@ -560,6 +560,17 @@ void RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const aux_sw
                 break;
             }
             break;
+
+        case AUX_FUNC::AIRMODE:
+            switch (ch_flag) {
+            case HIGH:
+                copter.ap.air_mode = AirModeEnabled;
+                break;
+            default:
```

Comment:
```suggestion
            case MIDDLE:
                break; // do nothing
            LOW:
```

Reply:
This allows MIDDLE to be used for something else later.

Generally I don't like `default:` statements, I've realised :-)

Reply:
Done